programs/perpetuals/src/state/oracle.rs

```diff
@@ -33,6 +33,7 @@ pub struct OraclePrice {
 pub struct OracleParams {
     pub oracle_account: Pubkey,
     pub oracle_type: OracleType,
+    pub permissionless_update_pubkey: Pubkey,
```

Comment:
lets rename this field, it is actually a public key of the oracle authority that will sign the message on a backend. A better name would be `oracle_authority`

Reply:
renamed 👍